src/components/Form/FormFactory.js

```diff
@@ -176,6 +176,19 @@ const onChange = (formFields, formValues, value, key, onFormChange) => {
   onFormChange({ value, validation }, key, formValid); // to update state data
 };
 
+const getLabel = field => (
+  <>
```

Comment:
I do not see a benefit of wrapping this in a Fragment

Reply:
It won't compile without it. How do you suggest it could be improved?

Reply:
seems to work fine here
```jsx
const getLabel = field =>
  field.type !== 'checkbox' && (
    <React.Fragment>
      <ControlLabel className={field.required ? 'required-pf' : null}>{field.title}</ControlLabel>
      {field.help && (
        <FieldLevelHelp className="kubevirt-form-group__field-help" placement="right" content={field.help} />
      )}
    </React.Fragment>
  );
```

Reply:
Done